lib/webdriver.py

```diff
@@ -201,6 +202,7 @@ def _wait_for_login(self, driver: Driver, account_monitor: AccountMonitor) -> No
             raise error
 
         self._set_account_name(account_monitor, login_response)
+        driver.click("div.nav-item.my-account-nav-item")
```

Comment:
Does this help the script stay undetected and fetch the reservations correctly or is this unnecessary?